.pre-commit-config.yaml

```diff
@@ -0,0 +1,102 @@
+    - id: check-merge-conflict
+    - id: detect-private-key
+- repo: https://github.com/hadolint/hadolint
+  rev: v2.10.0
+  hooks:
+    - id: hadolint
+- repo: https://github.com/alessandrojcm/commitlint-pre-commit-hook
+  rev: v8.0.0
+  hooks:
+    - id: commitlint
+      stages: [commit-msg]
+      additional_dependencies: ['@commitlint/config-angular']
+- repo: https://github.com/antonbabenko/pre-commit-terraform
+  rev: v1.72.1
+  hooks:
+    - id: terraform_fmt
+    - id: terraform_validate
+      files: ^examples/
+    - id: terraform_docs
+      args: ['--args=--hide=providers']
+    - id: terraform_tflint
+      args:
+        - --args=--enable-rule=terraform_unused_declarations
+        - --args=--enable-rule=terraform_deprecated_index
+        - --args=--enable-rule=terraform_comment_syntax
+        - --args=--enable-rule=terraform_documented_outputs
+        - --args=--enable-rule=terraform_documented_variables
+        - --args=--enable-rule=terraform_typed_variables
+        - --args=--enable-rule=terraform_naming_convention
+        - --args=--enable-rule=terraform_required_version
+        - --args=--enable-rule=terraform_required_providers
+        - --args=--enable-rule=terraform_unused_required_providers
+        - --args=--enable-rule=terraform_standard_module_structure
+    - id: terragrunt_fmt
+    - id: terragrunt_validate
+    - id: terraform_tfsec
+      files: '\.tf'
+    - id: terraform_checkov
+      args:
+        - --args=--skip-path=.scv
+        - --args=--skip-path=Dockerfile
+- repo: https://github.com/syntaqx/git-hooks
+  rev: v0.0.17
+  hooks:
+  - id: forbid-binary
+    exclude: .png|.jpg
+  - id: shellcheck
+  - id: go-fmt
+- repo: https://github.com/ibm/detect-secrets
+  rev: 0.13.1+ibm.50.dss
+  hooks:
+  - id: detect-secrets
+    args: ['--baseline', '.secrets.baseline', '--fail-on-unaudited', 'use-all-plugins']
+    exclude: go.sum
+# flake8
+- repo: https://github.com/pycqa/flake8
+  rev: 4.0.1
+  hooks:
+    - id: flake8
+      args: # arguments to configure flake8
+        # making isort line length compatible with black
+        - "--max-line-length=88"
+        # these are errors that will be ignored by flake8
+        # check out their meaning here
+        # https://flake8.pycqa.org/en/latest/user/error-codes.html
+        - "--ignore=E203,E501,W503"
+# isort
+- repo: https://github.com/pre-commit/mirrors-isort
+  rev: v5.10.1
+  hooks:
+  - id: isort
+    args: ["--profile", "black", "--filter-files"]
+# black
+- repo: https://github.com/ambv/black
+  rev: 22.3.0
+  hooks:
+  - id: black
+    args: # arguments to configure black
+      - --line-length=88
+- repo: local
+  hooks:
+    - id: golangci-lint
+      name: golangci-lint
+      description: Fast linters runner for Go.
+      entry: bash ci/golangci-lint.sh
+      types: [go]
+      language: golang
+      pass_filenames: false
+# helm lint
+- repo: https://github.com/gruntwork-io/pre-commit
+  rev: v0.1.17
+  hooks:
+    - id: helmlint
```

Comment:
delete this file, and use a symlink to the pre commit yaml in common-dev-assets

Reply:
fixed